app/connector/rest-swagger/src/main/resources/META-INF/syndesis/connector/rest-swagger.json

```diff
@@ -1,4 +1,5 @@
 {
+  "actions": [],
```

Comment:
Generated by `server-api-generator`

Reply:
it needs to be empty to please the check of connector-support-maven-plugin